docs/stats-engine/confidence-intervals.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Confidence Intervals
+sidebar_label: Confidence Intervals
+slug: /stats-engine/confidence-intervals
+---
+
+Confidence intervals are an intuitive way to quantify the uncertainty in the observed metric deltas.  A 95% confidence interval should contain the true effect 95% of the time.  This means that if we ran an experiment 100 times, the true value of the metric delta should be inside the confidence intervals 95 times.  
+
+In practical terms, a 95% confidence interval that doesn't contain zero represents a statstically significant result (with *&alpha; = 0.05*). Only 5% of the time would we expect to see that confidence interval if the true effect was zero.  Larger confidence intervals imply less certainity in the exact size of the effect with a larger range of likley values.  
```

Comment:
- Would an image/screenshot help here?  I was tripped up a bit by the "doesn't contain zero".
- aka false positive rate